# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: 1ae213bfa43e
# Full Hash: 1ae213bfa43e3270689641632f1a89e4c4affb8a
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2022-08-23 02:46:57
# Regressor Bug: 1781129
# File Overlap Count: 1
# Description:
#   Bug 1781129 - Part 3: Remove Shmem overload of GetSurfaceData, r=edgar
#   
#   After the previous changes there was only one consumer left of the Shmem
#   version of GetSurfaceData, which could easily be changed to use BigBuffer,
#   removing the need for that overload.
# ==============================================================================

diff -r dcebc98ea1f1 -r 1ae213bfa43e dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Mon Aug 22 15:38:15 2022 +0000
+++ b/dom/ipc/PBrowser.ipdl	Mon Aug 22 15:38:15 2022 +0000
@@ -636,7 +636,7 @@
     [Nested=inside_sync] sync DispatchTouchEvent(WidgetTouchEvent event);
 
     async InvokeDragSession(IPCDataTransfer[] transfers, uint32_t action,
-                            Shmem? visualData,
+                            BigBuffer? visualData,
                             uint32_t stride, SurfaceFormat format,
                             LayoutDeviceIntRect dragRect,
                             nsIPrincipal principal, nsIContentSecurityPolicy csp,